snowflake-cortex/README.md

```diff
@@ -0,0 +1,8 @@
+To run this example first execute this command:
+```bash
+snow sql -f 'prepare/prepare_data.sql'
```

Comment:
Should this be the shell file instead?

Reply:
The shell file is going to be used to load data into the consumer side